Godeps/_workspace/src/github.com/whyrusleeping/go-multiplex/.gxlastpubver

```diff
@@ -0,0 +1 @@
+QmYmW76Y7NxvWwW3GPUmpbHd4mm1iW14f5UimeJ9UoofEs
```

Comment:
@ipfsbot pin QmYmW76Y7NxvWwW3GPUmpbHd4mm1iW14f5UimeJ9UoofEs

o/ that should work